paws-of-luv-frontend/src/App.css

```diff
@@ -1,13 +1,74 @@
+/* ########## B O D Y ########## */
+body {
```

Comment:
important tag was important, ha

Reply:
Who would have known. That's reactstrap for ya.